src/constants/phone-types.js

```diff
@@ -0,0 +1,4 @@
+module.exports = Object.freeze({
```

Comment:
Yeah I think this would be a big improvement. There are quite a lot of places where we manually type out the journey event or page name which should probably be constants in reality

Reply:
Would it be preferable to have one constants file as opposed to one for each type?

Reply:
Maybe, I think we'll find out once we start putting other constants here.